nntoolkit/create.py

```diff
@@ -101,4 +109,4 @@ def main(nn_type, architecture, model_file):
 
 if __name__ == '__main__':
     args = get_parser().parse_args()
-    main(args.type, args.architecture, args.model_file)
+    main(args.type, args.architecture, args.model_file)
```

Comment:
This newline should be here (see http://www.reddit.com/r/Python/comments/1zjugg/question_why_does_pep8_recommend_leaving_a_blank/)